tests/rptest/utils/rpenv.py

```diff
@@ -23,3 +23,19 @@ def sample_license():
         assert is_ci == "false"
         return None
     return license
+
+
+class IsCIOrNotEmpty:
+    """
+    Comparison with this object is true if the environemnt is CI, or if the
+    other value is not empty.
+
+    Useful as a markup on a test that must run in CI or when the env is present:
+
+    @env(REQUIRED_ENV=IsCIOrNotEmpty())
+    """
+    def __init__(self):
+        self.is_ci = not os.environ.get("CI", "false")
```

Comment:
So the `CI` environmental variable is only present when we're doing CI?  Like we don't have `CI=false` for non-CI builds? I think this will always return false?

I ran a test in python where I set `TEST_ENV` to `true` and did not set anything for `BLAH`:

```python
>>> os.environ.get("TEST_ENV", "false")
'true'
>>> not os.environ.get("TEST_ENV", "false")
False
>>> os.environ.get("BLAH", "false")
'false'
>>> not os.environ.get("BLAH", "false")
False
```

Reply:
In CI, CI='true', default is 'false'. I've made this simpler.